Commands/IdentityCommand.py

```diff
@@ -0,0 +1,59 @@
+from discord.ext import commands
+
+from Commands.SafeOAuthBasedCommand import SafeOAuthBasedCommand
+
+
+class IdentityCommand(SafeOAuthBasedCommand):
+    """
+    `IdentityCommand` is a class conforming to `SafeOAuthBasedCommand` that allows users to create a profile for
+    themselves so that they can register for raids.
+    """
+
+    def __init__(self, identity_worksheet, list_available_classes, list_available_roles):
+        """
+        Initializes an `IdentityCommand` with the provided classes and roles that are available.
+
+        :param identity_worksheet: The worksheet where user identities should be stored to
+        :param list_available_classes: The list of available classes that a user can choose from
+        :param list_available_roles: The list of available roles that a user can choose from
+        """
+        self.__identity_worksheet = identity_worksheet
```

Comment:
Initializing and pointing to the worksheet is fine, but I don't think it's necessary to pipe int he available classes and roles. Let's just make those available as package constants somewhere else, since they are presumably usable in a few different domains.